src/JsonSchema/Validator.elm

```diff
@@ -62,6 +68,36 @@ validateItems items values =
         Just itemSchema ->
             List.concat (List.indexedMap (\i v -> List.map (appendName (toString i)) (validate itemSchema v)) (Array.toList values))
 
+validateTupleItems : Maybe (List Schema) -> Array.Array Value -> List Error
+validateTupleItems items values =
+    case items of
+        Nothing ->
+            []
+
+        Just itemSchemas ->
+            let
+                validateWithIndex i itemSchema v =
+                    validate itemSchema v
+                        |> List.map (appendName (toString i))
+            in
+
+                {-- Note: the spec doesn't specify what should happen if you
+                 have more schemas than values. If you have less schemas, it
+                 suggests the extra values won't be validated.
```

Comment:
Ah, that answers my previous question about how `maxItems` and `minItems` are supposed to work!